WordPressKit/WordPressComRestApi.swift

```diff
@@ -268,11 +268,11 @@ open class WordPressComRestApi: NSObject {
      - parameter success:    callback to be called on successful request
      - parameter failure:    callback to be called on failed request
 
-     - returns:  a NSProgress object that can be used to track the progress of the upload and to cancel the upload. If the method
+     - returns:  a `Progerss` object that can be used to track the progress of the upload and to cancel the upload. If the method
      returns nil it's because something happened on the request serialization and the network request was not started, but the failure callback
      will be invoked with the error specificing the serialization issues.
      */
-    @objc @discardableResult open func multipartPOST(_ URLString: String,
```

Comment:
We could add a `@nonobj` to this function here, so that it won't be bridged to Objective-C, which will conflicts with the other almost exact function. However, if that ever happens for some reason, the compiler probably would warn us, even without `@nonobj`. Maybe I worry too much...

Given this function a `@nonobjc` declaration also convey that the other `multiparPOST` is for Objective-C, not this one.

Reply:
> TIL "Plain Old NSObject" 😄

TIL `@nonobjc` 😄 

🙌

Reply:
Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/765/commits/d7d0bf74194bc7953eaa4dc36a87eab85f06a276.

Auto-merged enabled. Then I'll ship version 15.0.0 as discussed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/767#issuecomment-2019316612